### DISCUSSION TITLE
I want to use as french

### DISCUSSION THREAD

**Discussion by ScreamZ** [shuding/nextra#1490] [Q&A]
Hello,

I would like to get a french website, but I don't need to support multiple languages. Is there any way to translate statics parts for doc template ? Easily ?

Or how do I proceed ?

Thanks

**Comment by ImBIOS** [comment]
Just use Frnench in your docs/blog. It's possible to not use multiple language support. Currently, we need to create a whole markdown file to translate the page, it should be the best option for translating docs/blogs. Because we always need to translate the whole text for each page.
